routes/items.js

```diff
@@ -18,14 +28,13 @@ router.post("/", (req, res, next) => {
 });
 
 router.put("/:id", (req, res, next) => {
-  // TODO: complete writing this route handler
-  res.status(501).send({ error: 'route not yet implemented' });
+  itemData.updateById(req.params.id, req.body);
+  res.status(200).send();
 });
 
-
 router.delete("/:id", (req, res, next) => {
-  // TODO: complete writing this route handler
-  res.status(501).send({ error: 'route not yet implemented' });
+  itemData.deleteById(req.params.id);
+  res.status(200).send();
```

Comment:
For cases where a status is all that's returned, you can use the `sendStatus()` method rather than chaining `status()` and `send()`.